---
title: "Learning protein constitutive motifs from sequence data"
collection: publications
permalink: /publication/2019-03-01-Learning-protein-constitutive-motifs-from-sequence-data
excerpt: 'Publisher: eLife Sciences Publications, Ltd'
date: 2019-03-01
venue: 'eLife'
paperurl: 'https://doi.org/10.7554/eLife.39397'
paperpdf: '/files/Tubiana et al. - 2019 - Learning protein constitutive motifs from sequence.pdf'
citation: ' Jérôme Tubiana,  Simona Cocco,  Rémi Monasson, &quot;Learning protein constitutive motifs from sequence data.&quot; eLife, 2019.'
---
<b> Abstract: </b>Statistical analysis of evolutionary-related protein sequences provides information about their structure, function, and history. We show that Restricted Boltzmann Machines (RBM), designed to learn complex high-dimensional data and their statistical features, can efficiently model protein families from sequence information. We here apply RBM to 20 protein families, and present detailed results for two short protein domains (Kunitz and WW), one long chaperone protein (Hsp70), and synthetic lattice proteins for benchmarking. The features inferred by the RBM are biologically interpretable: they are related to structure (residue-residue tertiary contacts, extended secondary motifs (α-helixes and β-sheets) and intrinsically disordered regions), to function (activity and ligand specificity), or to phylogenetic identity. In addition, we use RBM to design new protein sequences with putative properties by composing and &apos;turning up&apos; or &apos;turning down&apos; the different modes at will. Our work therefore shows that RBM are versatile and practical tools that can be used to unveil and exploit the genotype–phenotype relationship for protein families.
